### PR TITLE
View Profile Use Case

### PR DESCRIPTION
**Pull request by anbu1504**
1. Implemented the view profile use case
2. Displays username, name, email, and rating
3. Has logout and back button
4. Tests are implemented

### CODE REVIEW

**Comment by YathusanK** [comment]
According to our UI flow, the view profile screen should also show the projects of the user as well as collaboration requests.

**Review by YathusanK — CHANGES_REQUESTED** [review]

File: src/main/java/data_access/ViewProfileDataFileAccessObject.java
[the remark this answers is not in the document]
Comment (YathusanK) [reply src/main/java/data_access/ViewProfileDataFileAccessObject.java:1]: This file is actually used for testing the interactor in isolation from the actual database.

File: src/main/java/view/ViewProfileView.java [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        );*/
+        setLayout(new GridBagLayout());
+        GridBagConstraints c = new GridBagConstraints();
+        JPanel viewProfileOutputData = create();
+
+
+        addComponent(usernamePanel, c, 0, 0, GridBagConstraints.HORIZONTAL);
+        addComponent(namePanel, c, 0 , GridBagConstraints.RELATIVE, GridBagConstraints.HORIZONTAL);
+        addComponent(emailPanel, c, 0, GridBagConstraints.RELATIVE, GridBagConstraints.HORIZONTAL);
+        addComponent(ratingsPanel, c, 0, GridBagConstraints.RELATIVE, GridBagConstraints.HORIZONTAL);
```
Comment (Pranjal-R-Agrawal) [inline src/main/java/view/ViewProfileView.java:111]: We aren't implementing the rating system anymore, so this should be removed / commented out

**Review by Tanmay-Shinde — APPROVED** [review]
Great work, all commits look good. ready to merge.

**Review by YathusanK — APPROVED** [review]
Good work! Your changes fixed the earlier mentioned issues. Your code also adheres Clean Architecture and it functions correctly.

**Pull request merged**